plugins/woocommerce/src/Internal/Admin/Suggestions/PaymentExtensionSuggestions.php

```diff
@@ -3412,4 +3359,86 @@ private function standardize_extension_details( array $extension_details ): arra
 
 		return $standardized;
 	}
+
+	/**
+	 * Based on the WC onboarding profile, determine if the merchant is selling online.
+	 *
+	 * If the user skipped the profiler (no data points provided), we assume they are selling online.
+	 *
+	 * @return bool True if the merchant is selling online, false otherwise.
+	 */
+	private function is_merchant_selling_online(): bool {
```

Comment:
I'm assuming those were just moved to the bottom? It's the same implementation as before, right? Is there any reason for moving these functions?

Reply:
Yes, no changes, it was moved to the end of class for a better grouping.